src/elm/CompoundComponents/Eth/ConnectedEthWallet.elm

```diff
@@ -569,8 +582,10 @@ chooseWalletView userLanguage isCompoundChain ({ chooseWalletState } as model) =
                          ]
                             ++ headerDescriptions
                             ++ [ div [ class "connect-choices" ]
-                                    ([ connectItemView userLanguage isCompoundChain Metamask
-                                     ]
+                                     (
+                                           [ connectItemView userLanguage isCompoundChain Metamask]
+                                        ++ lineDivider   
+                                        ++  [ connectItemView userLanguage isCompoundChain Tally]
```

Comment:
Last comment I swear. Let's go head and move Tally to the end of the current list. Right now the order is by most popular and am happy to re-order in the future as tally gains traction.

Reply:
Np. I totally understand. We try to put higher up to help with optics and traction. It has been moved and I appreciate all the feedback.